### DISCUSSION TITLE
customize node modules

### DISCUSSION THREAD

**Discussion by Prashant412** [cloudscape-design/components#2310] [Q&A] [answered]
i have customize all the cloudscape component to my way but when using in git node module isnt share and after installing it again my node module changed to orignal what to to

**Comment by rubencarvalho** [comment, marked as answer]
Hello,

I am not sure I fully understand your question, however it seems to me you are trying to customize the look and feel of the components. We do not support CSS overrides but we have a first party API for this, which you can read more about on the [Theming](https://cloudscape.design/foundation/visual-foundation/theming/) page of our documentation website.

`node_modules` is meant to store dependencies as they are installed from `npm`, and any changes made directly to these files will be lost when the modules are reinstalled or updated.

Let me know if that helps!

Thank you.
